--- lxpanel-0.5.10/src/plugins/netstatus/netstatus-iface.c.str_fmt~	2012-10-24 07:33:51.749978454 +0200
+++ lxpanel-0.5.10/src/plugins/netstatus/netstatus-iface.c	2012-10-24 07:33:59.767098520 +0200
@@ -408,7 +408,7 @@ netstatus_iface_set_polling_error (Netst
   va_start (args, format);
 
   error_message = g_strdup_vprintf (format, args);
-  error = g_error_new (NETSTATUS_ERROR, code, error_message);
+  error = g_error_new (NETSTATUS_ERROR, code, "%s", error_message);
 
   dprintf (POLLING, "ERROR: %s\n", error->message);
   netstatus_iface_set_error (iface, error);
